wrapers.py

```diff
@@ -0,0 +1,34 @@
+import jwt
+from flask import request, jsonify, current_app
+from functools import wraps
+
+
+from db import User
+
+
+def token_required(f):
```

Comment:
To make it more readable, it might be worth replacing `f` with `func`, for example.

Reply:
https://github.com/mstoro/nasa_api/pull/1/commits/606c73d5d0205ad63bdbcc79471c67f249994c86